src/components/__tests__/Tooltip.test.tsx

```diff
@@ -188,17 +199,18 @@ describe('Tooltip', () => {
             const currentProps = tooltip.props
             const shouldUpdate = tooltip.shouldComponentUpdate(
                 currentProps,
-                currentState
+                currentState,
+                undefined
             )
             expect(shouldUpdate).toBe(false)
         })
     })
 
     it('sets ref of tooltip and wrapper after mounting', () => {
-        const tooltip = mount(getTooltip()).instance()
+        const tooltip = mount(getTooltip()).instance() as Tooltip
 
-        expect(tooltip.wrapper).toBeTruthy()
-        expect(tooltip.tooltip).toBeTruthy()
+        expect((tooltip as any).wrapper).toBeTruthy()
```

Comment:
I think in tests it makes more sense to disable checking for the entire file than do to incorrect casts.

Reply:
I really don't want to do this as it's just a few lines that are being casted here, and we lose type safety for the entire file because of that. I'll try to rework the Tooltip file to get proper types. I tried to go down that rabbit hole, but Tooltip has a dependency on Popover etc, and then I end up reworking a lot more files than I initially wanted to be in scope.

Reply:
You can use @ts-ignore or @ts-expect-error (better) so that it works without introducing incorrect types.